x/batch/msg.go

```diff
@@ -19,7 +19,7 @@ type Msg interface {
 func Validate(msg Msg) error {
 	l, err := msg.MsgList()
 	if err != nil {
-		return errors.Wrap(err)
+		return errors.Wrap(err, "error retrieving batch messages")
```

Comment:
I think "cannot retrieve batch message" might be a better phrasing. :nail_care:

Reply:
I agree with these naming suggestions, also above.
If we return an error, the text "error" is pretty unnecessary....
"error file not found" or "file not found"?

Reply:
yeah, same, this was just a placeholder to make the whole refactoring work